larray/core/array.py

```diff
@@ -111,13 +101,11 @@
 except ImportError:
     np_nanprod = None
 
-from larray.oset import *
-from larray.utils import (table2str, size2str, unique, csv_open, long,
-                          decode, basestring, unicode, bytes, izip, rproduct,
-                          ReprString, duplicates, array_lookup2, strip_rows,
-                          skip_comment_cells, find_closing_chr, StringIO, PY2,
-                          float_error_handler_factory)
-
+from larray.util.utils import (table2str, size2str, unique, decode, basestring, izip, rproduct, ReprString, StringIO,
+                               duplicates, float_error_handler_factory)
+from .axis_group import (Axis, AxisCollection, x, Group, PGroup, LGroup, ExprNode, remove_nested_groups, _isnoneslice,
```

Comment:
I would rather use absolute import like other lines (or convert them all to relative)